mantidproject/mantidimaging
# Add load sample log as an option

Fixes #688 

To test:
- Load data without a sample log
- Click File > Load log for stack...
- Select the tomo stack
- Select a sample log
- Check that monitor normalization in the operations window is performed normally
